aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogCommitFailure.java

```diff
@@ -426,10 +408,8 @@ public void testS3Exception() {
 
     GlueTableOperations spyOps = Mockito.spy(ops);
     failCommitAndThrowException(spyOps, S3Exception.builder().statusCode(300).build());
-
-    AssertHelpers.assertThrows(
-        null, S3Exception.class, () -> spyOps.commit(metadataV2, metadataV1));
-
+    Assertions.assertThatThrownBy(() -> spyOps.commit(metadataV2, metadataV1))
+        .isInstanceOf(S3Exception.class);
```

Comment:
same here, we should add a check against the message

Reply:
I think here it's not needed as the exception without a message is thrown from the above line

Reply:
if there's no message, then we should still add a check for whatever the message is (null, empty)

Reply:
Thanks for the suggestion. Added